lib/requester/core.js

```diff
@@ -289,6 +291,20 @@ var dns = require('dns'),
         // - localhost
         // - *.localhost
         if (getTLD(lowercaseHost) !== LOCALHOST) {
+            // When options.all is set, the callback expects array of addresses instead of a single address
+            if (options.all) {
```

Comment:
was this breaking till now ?

Reply:
No. Only after node version 20. Reason in RCA - https://postmanlabs.atlassian.net/wiki/spaces/AT/pages/4532502534/APITEST-375+Localhost+not+resolving+in+postman-runtime+for+node+version+20